public/app/features/dashboard-scene/saving/provisioned/hooks.ts

```diff
@@ -1,3 +1,6 @@
+import Chance from 'chance';
```

Comment:
```suggestion
import { Chance } from 'chance';
```
I thin it'd be this?